src/Main/CommentAdder.jsx

```diff
@@ -0,0 +1,65 @@
+import { useState } from "react";
+import { postComment } from "../utils/api";
+
+export const CommentAdder = ({ article_id, setComments, loggedInUser }) => {
+  const [newComment, setNewComment] = useState("");
+  const [postInProgress, setPostInProgress] = useState(null);
+  const [postSuccess, setPostSuccess] = useState(null);
+  const [emptyComment, setEmptyComment] = useState(null);
+  const [err, setErr] = useState(null);
+
+  const handleSubmit = (event) => {
```

Comment:
Looks good but I think the submit of the form can be clicked again mid-post request and acan also post the same comment multiple times accidently if the text box is not cleared.